doc(release) Release notes modules 24.04 september 2025 release (#4686)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-commercial-extensions.mdx

## Centreon MAP

### 24.04.16

Release date: `September 26, 2025`

> <details>
>   <summary>Bug fixes</summary>
> 
> - [Config] The configure.sh script does not start the mysql service anymore.
> - [Editor] Fixed an issue that caused links to be positioned differently in the editor when migrating from MAP Legacy to MAP.	
> backport.
> - [Editor] Fixed issues when copy/pasting elements.
> - [MAP Editor] Fixed an issue causing parent name to be wrong in tooltip after saving.
> - [Upgrade] remove mention of generate_routes.php.
> - [Viewer] Business Activities - Fixed visualisation of status and state of KPIs.
> - [Viewer] Fixed an issue preventing data from being displayed when hovering on a meta-service.
> - [Viewer] Fixed an issue that prevented pie chart widget from being displayed in percentage per status.
> - [Viewer] Fixed an issue where BAs linked to a remote server were not displayed.
> 
> </details>
> 
> <details>
>   <summary>Security fixes</summary>
> 
> - [Server] Fixed ownership of configuration files.
> - [Vulnerability] Upgraded crypto-js to version 4.2.0.
> - [Vulnerability] Upgraded Tomcat to version 10.1.44.
> 
> </details>

Reviewer: ```suggestion
<details>
  <summary>Bug fixes</summary>

- [Config] Fixed an issue where the configure.sh script was not starting the mysql service anymore.
- [Editor] Fixed an issue that caused links to be positioned differently in the editor when migrating from MAP Legacy to MAP.	
- [Editor] Fixed issues occurring when copying/pasting elements.
- [MAP Editor] Fixed an issue causing incorrect parent names to appear in tooltips after saving.
- [Upgrade] Removed mention of generate_routes.php.
- [Viewer] Business Activities: fixed visualization of status and state of KPIs.
- [Viewer] Fixed an issue preventing data from being displayed when hovering over a metaservice.
- [Viewer] Fixed an issue that prevented pie chart widgets from being displayed in percentage per status.
- [Viewer] Fixed an issue where BAs linked to a remote server were not displayed.

</details>

<details>
  <summary>Security fixes</summary>

- [Server] Fixed ownership of configuration files.
- [Vulnerability] Upgraded crypto-js to version 4.2.0.
- [Vulnerability] Upgraded Tomcat to version 10.1.44.

</details>
```

## Centreon BAM

### 24.04.10

Release date: `September 29, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Upgrade] remove mention of generate_routes.php.

Reviewer: ```suggestion
- [Upgrade] Removed mention of generate_routes.php.
```

## Centreon MBI

### 24.04.9

Release date: `September 26, 2025`

<details>
  <summary>Enhancements</summary>

- [CBIS] Versions 2 and 3 of SMB are now supported.

</details>

<details>
  <summary>Bug fixes</summary>

- [ETL] Fixed an issue occurring when metric names exceeded 255 characters.

> - [MBI] Fixed an issue causing ACLs not to be applied on report designs.

Reviewer: ```suggestion
- [MBI] Fixed an issue causing ACLs not to be applied to report designs.
```

## Centreon MBI

### 24.04.9

Release date: `September 26, 2025`

<details>
  <summary>Enhancements</summary>

- [CBIS] Versions 2 and 3 of SMB are now supported.

</details>

<details>
  <summary>Bug fixes</summary>

> - [ETL] Fixed issue when metric names exceed 255 characters.

Reviewer: ```suggestion
- [ETL] Fixed an issue occurring when metric names exceeded 255 characters.
```

> - [MBI] Fixed an issue causing ACLs not to be applied on Report designs.

Reviewer: ```suggestion
- [MBI] Fixed an issue causing ACLs not to be applied on report designs.
```

- [MBI] Fixed upload of report templates.

> - [Upgrade] remove mention of generate_routes.php.

Reviewer: ```suggestion
- [Upgrade] Removed mention of generate_routes.php.
```

## Centreon Auto Discovery

### 24.04.6

Release date: `September 29, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Upgrade] remove mention of generate_routes.php.

Reviewer: ```suggestion
- [Upgrade] Removed mention of generate_routes.php.
```

## Centreon Anomaly Detection

### 24.04.7

Release date: `September 29, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Upgrade] remove mention of generate_routes.php.

Reviewer: ```suggestion
- [Upgrade] Removed mention of generate_routes.php.
```

## Centreon IT Edition Extensions

### 24.04.7

Release date: `September 29, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Upgrade] remove mention of generate_routes.php.

Reviewer: ```suggestion
- [Upgrade] Removed mention of generate_routes.php.
```
